### ISSUE TITLE
roosterteeth.com support

### ISSUE THREAD

**Issue by ultimatejacob27** [FrostCo/AdvancedProfanityFilter#499]
_Instructions: This template is written in Markdown. Click the "Preview" tab to see what it will look like when submitted. Fill in the applicable sections (you can erase the examples if you'd like). For more information, please see [this page](https://github.com/FrostCo/AdvancedProfanityFilter/wiki/New-Audio-Muting-Site)._

#### :movie_camera: The Site
_Example: www.roosterteeth.com_

#### :speech_balloon: Subtitle Element/Node
_Example:_
```html
<div style="position: absolute; inset: 0px; margin: 1.5%;"><div style="direction: ltr; writing-mode: horizontal-tb; unicode-bidi: plaintext; text-align: center; font: 25.29px sans-serif; white-space: pre-line; position: absolute; width: 997.788px; inset: 476.15px 0px 0px; height: 29.6px; background-color: rgba(0, 0, 0, 0);" class="vjs-text-track-cue vjs-text-track-cue-en" lang="en"><div style="color: rgb(255, 255, 255); background-color: rgb(0, 0, 0); position: relative; inset: 0px; display: inline; writing-mode: horizontal-tb; unicode-bidi: plaintext; font-family: sans-serif;">We're all facing Salem together.</div></div></div>
```

#### :pencil: Additional notes
_Add any other context or screenshots about the request here._

**Comment by richardfrost** [comment]
Hi @ultimatejacob27, thanks for the suggestion.

Could you give this custom config a try?

```json
{
  "roosterteeth.com": [
    {
      "mode": "elementChild",
      "parentSelector": "div.vjs-text-track-display",
      "subtitleSelector": "div.vjs-text-track-cue > div",
      "tagName": "DIV"
    }
  ]
}
```

To use this config, paste it into the Custom Sites area (Options page > Audio tab) and click the blue "SAVE" button underneath it. Then, make sure to refresh the tab for the video site (if you have it open still) and it will load this new configuration.

For best results, make sure to remove any custom config used for testing like this once its been included in the official update.

**Comment by ultimatejacob27** [comment]
That worked perfectly, thanks!

**Comment by richardfrost** [comment]
Thanks for reporting back @ultimatejacob27. I'm glad it is working for you! I'll include the site in the next version of the filter. Take care!

**Comment by richardfrost** [comment]
@ultimatejacob27 

This has been included in version [3.2.1](https://github.com/richardfrost/AdvancedProfanityFilter/releases/tag/3.2.1). Feel free to comment here if you have any feedback for this update.
